listenbrainz/webserver/errors.py

```diff
@@ -57,41 +57,61 @@ def error_wrapper(template, error, code):
         resp.headers['Access-Control-Allow-Origin'] = '*'
         return resp, code
 
+    def json_error_wrapper(error, code):
+        return jsonify({
+            'code': code,
+            'error': error.description,
+            }), code
+
+    def handle_error(error, code):
+        """ Returns appropriate error message on HTTP exceptions
+
+            error (werkzeug.exceptions.HTTPException): The exception that needs to be handled
+            code (int): the HTTP error code that should be returned
+
+            Returns:
+                A Response which will be a json error if request was made to the LB api and an html page
+                otherwise
+        """
+        if request.path.startswith('/1/'):
```

Comment:
I'm wondering if we could future-proof this or at least get rid of some magic strings.
I'm not sure if there was a reason why our API blueprint is not mounted with a url_prefix: https://github.com/paramsingh/listenbrainz-server/blob/538c0a5ac12dfb72c7a6cfaad6f8cda9b0947962/listenbrainz/webserver/__init__.py#L109
If there was, we could have a const in settings which we can use here.